### PR TITLE
README updates with missing details for OVA process

### PR DESCRIPTION
**Pull request by miyamotoh**
Just went through the steps for preparing, uploading, and importing a new OVA, and noticed a few missing info in `README.md` that could make following this doc a little smoother, based on errors that I encountered (on RHEL8).

Signed-off-by: Hiro Miyamoto <miyamotoh@us.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/17#" title="Author self-approved">miyamotoh</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **bpradipt**
You can assign the PR to them by writing `/assign @bpradipt` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: scripts/images/README.md
Comment (mkumatag) [inline scripts/images/README.md:20]: few instructions are already there in the python file and in the help message as well, having that said - we can have here as well.

File: scripts/images/README.md
Comment (mkumatag) [inline scripts/images/README.md:53]: It will be set for the OS password for the image

**Review by mkumatag — CHANGES_REQUESTED** [review]
thanks for the PR, please address those minor comments

File: scripts/images/README.md
Comment (miyamotoh) [reply scripts/images/README.md:20]: Glad to hear you agree to have these upfront for smoother operation, instead of getting error and having to hunt for fix.

File: scripts/images/README.md
Comment (miyamotoh) [reply scripts/images/README.md:53]: Will drop this change in my next push. 👍

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/17#" title="Author self-approved">miyamotoh</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/17#issuecomment-699501555" title="Approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
